### PR TITLE
A few small fixes

### PR DESCRIPTION
**Pull request by mwlang**
* handle parsing timestamps when value not supplied (return nil instead of exception).
* handle open and closed order on the Order class.
* gracefully handle a symbol not returning data for Quote class.
* additional attributes for Order class.

Specs updated.  Plus, can see code in action on my [crypto project](https://github.com/mwlang/crypto)

### CODE REVIEW

File: lib/bittrex.rb [hunk 1]
```diff
@@ -12,6 +12,11 @@ module Bittrex
   autoload :Wallet,        'bittrex/wallet'
   autoload :Withdrawal,    'bittrex/withdrawal'
 
+  def self.extract_timestamp value
```
Comment (mwerner) [inline lib/bittrex.rb]: Please follow the declaration styling of the rest of the repo `def self.extract_timestamp(value)`

File: lib/bittrex/deposit.rb [hunk 2]
```diff
@@ -9,7 +9,7 @@ def initialize(attrs = {})
       @quantity = attrs['Amount']
       @currency = attrs['Currency']
       @confirmations = attrs['Confirmations']
-      @executed_at = Time.parse(attrs['LastUpdated'])
+      @executed_at = Bittrex::extract_timestamp(attrs['LastUpdated'])
```
Comment (mwerner) [inline lib/bittrex/deposit.rb]: Please invoke this off the class directly: `Bittrex.extract_timestamp`. Or preferably, move `extract_timestamp` into a helper module so the method isn't on the parent namespace eg `Bittrex::Helpers.extract_timestamp`

File: lib/bittrex/order.rb [hunk 3]
```diff
@@ -14,9 +14,11 @@ def initialize(attrs = {})
       @total = attrs['Total']
       @fill = attrs['FillType']
       @limit = attrs['Limit']
-      @commission = attrs['Commission']
+      @commission = (attrs['Commission'] || attrs['CommissionPaid']).to_f
```
Comment (mwerner) [inline lib/bittrex/order.rb]: please guard against a nil value here

Comment (mwlang) [reply lib/bittrex/order.rb]: Can you explain this one further?  It is guarding against nil values.  @commission is guaranteed to be set to 0.0 if both attributes are nil.

File: lib/bittrex.rb (hunk 1 above)
Comment (mwlang) [reply lib/bittrex.rb]: done!

File: lib/bittrex/deposit.rb (hunk 2 above)
Comment (mwlang) [reply lib/bittrex/deposit.rb]: Extracted to helper module.

File: lib/bittrex/order.rb (hunk 3 above)
Comment (mwerner) [reply lib/bittrex/order.rb]: got it. didn't catch the `to_f` on the end

**Comment by CryptoKlizO** [comment]
WHAt is this
a few small fixes ?

is this me being hacked?

-  CryptOKlizO

**Comment by CryptoKlizO** [comment]
Hello wtf?

-  CryptOKlizO

**Pull request merged**